pkg/agent/controller/networkpolicy/l7engine/reconciler.go

```diff
@@ -178,7 +178,7 @@ func generateTenantRulesData(policyName string, protoKeywords map[string]sets.Se
 	// Refer to Suricata detect engine in codebase for detailed tag keyword configuration.
 	var tagKeyword string
 	if enableLogging {
-		tagKeyword = " tag: session, 30, seconds;"
+		tagKeyword = " tag:host;"
```

Comment:
the comment above is outdated, as it still says "the session is tagged for 30 seconds"

Reply:
Updated.